vega/widget.py

```diff
@@ -115,21 +126,36 @@ def opt(self, value):
         self._opt_source = json.dumps(value)
         self._reset()
 
-    def update(self, key, remove=None, insert=None):
+    def update(self, key, remove=None, insert=None, resize=True):
         """Update the chart data.
 
         Updates are only reflected on the client, i.e., after re-displaying
         the widget will show the chart specified in its spec property.
 
+        :param str key:
+            the name of the dataset to update, as declared in the data
+            section of the spec.
+
         :param Optional[str] remove:
             a JavaScript expression of items to remove. The item to test can
             be accessed as ``datum``. For example, the call
             ``update(remove="datum.t < 5")`` removes all items with the
             property ``t < 5``.
 
         :param Optional[List[dict]] insert:
```

Comment:
Does this assume that we only support a single dataset? I think we should support multiple datasets.